pkgs/development/compilers/fasmg/bin.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
```

Comment:
You can use `fetchzip` with `postFetch` instead of `stdenvNoCC.mkDerivation`, since all you need is just copy downloaded files to `$out`. For example, see https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/norwester/default.nix.

Reply:
I could remove fasmg-bin from `all-packages.nix` and `fasmg/bin.nix`, and just use a `let in` in `fasmg/default.nix`.
Would it be better?